CMakeLists.txt

```diff
@@ -21,6 +21,7 @@ if(CMAKE_Fortran_COMPILER_ID STREQUAL GNU)
   endif()
   add_compile_options(-fimplicit-none)
   add_compile_options(-ffree-line-length-132)
+  add_compile_options(-fno-range-check)
```

Comment:
See my comment in the documentation about this -- if it isn't needed for gfortran 9+ it might be better not to have this -- otherwise if it is needed, we should consider whether it needs documenting in the README (as it is easy to compile stdlib with user-specified flags, which could miss this).

Reply:
It is required for version 9.